### PR TITLE
Only use SSL for Google Analytics (closes #1578).

### LINKED ISSUE #1578: Get our Google Analytics on SSL
In our CSP we allowlist www.google-analytics.com.  I noticed AMO is allowing https://ssl.google-analytics.com.  I like that https restriction.  Let's do that for us too -- why are they using ssl.\* instead of www.* and how do we do that?

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @clouserw

### CODE REVIEW

File: testpilot/settings.py [hunk 1]
```diff
@@ -357,8 +357,7 @@ def lazy_langs():
     'https://*.mozilla.org',
     'http://*.gravatar.com',
     'https://*.gravatar.com',
-    'http://*.google-analytics.com',
-    'https://*.google-analytics.com',
+    'https://ssl.google-analytics.com',
```
Comment (chuckharmston) [inline testpilot/settings.py:363]: Probably not necessary to change this file, but meh.

**Comment by chuckharmston** [comment]
Worth noting that the file from `ssl.google-analytics.com` is hardcoded to use `//www.google-analytics.com` for pings, but since `http://ssl.google-analytics.com` redirects to `https://ssl.google-analytics.com`, it will always use `https`. Accordingly, I've updated `script-src` to only use `https://ssl.` in `script-src`, but still allow `https://www.` in `img-src` and `connect-src`.

File: testpilot/settings.py (hunk 1 above)
Comment (lmorchard) [reply testpilot/settings.py:363]: 😐 Django should be excised soon

**Review by lmorchard — APPROVED** [review]
LGTM FWIW but don't want to preempt a clouserw review

**Review by ghost — APPROVED** [review]
Nice, this looks a lot more secure.

**Comment by chuckharmston** [comment]
🔒 💚

File: testpilot/settings.py (hunk 1 above)
Comment (chuckharmston) [reply testpilot/settings.py:363]: https://github.com/mozilla/testpilot/issues/1608

**Comment by ghost** [comment]
Marking needs QA as I think is the convention now.  We want to verify this on stage.  That can be done by basically looking at the console and making sure there aren't any CSP errors.  Extra credit if you can see pings going to google analytics in the network tab.  Thanks!

/cc @SoftVision-PaulOiegas

**Comment by SoftVision-PaulOiegas** [comment]
@wresuolc @chuckharmston Guys, was this deployed on staging or not ? 
I'm asking because since morning we tried to test this on staging and we haven't got any "POST" or "GET" GA events with "https://ssl.google-analytics.com" url. Also no CSP errors displayed while navigating trough Test Pilot pages, only when external pages like GitHub and some footer links were opened.

However, we also tried the same tests on dev server where we found lots of "POST" google analytics collect events with SSL in url but none of the "GET" collect ones had it. If we remember well the "ssl" change was not present yesterday on dev, correct me if I'm wrong. 
So we switched testing on dev server since we found the "ssl" change there and looked also for CSP errors. The problem is that almost every GA event displays a CSP error on dev server. We have seen them for navigating on experiments pages, experiments enable/disable actions and also for tour navigation.

Did we wrongly understood how the events should be displayed or there is an issue here ?

**Stage console**
browser console on stage

**Dev console**
browser console on dev

**Comment by ghost** [comment]
@SoftVision-PaulOiegas sorry for the misunderstanding.  I just meant, when we push to stage at our normal time we should check this.  Stage was not updated.

Additionally, http://testpilot.dev.mozaws.net/ is out of date and testpilot-static.dev.mozaws.net doesn't have the nginx proxy in front of it (which we need to test this).  So, we don't have a working dev server right now apparently.  https://github.com/mozilla/testpilot/issues/1522 should fix that.
